code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -1,1002 +1,1003 @@
-#define AI_CHECK_WIRELESS 1

```

Comment:
this file also looks weird, some kind of whitespace or encoding changes

Reply:
I think this one is to fix the file to bring it into correct whitespace.   However, asking that be done in a separate PR would be a reasonable argument.

Reply:
Gonna commit with the fixed endings.